app/views/homes/show.html.erb

```diff
@@ -1,23 +1,26 @@
 <%= page_title "Home" %>
+
 <% add :about, "This page lets you quickly get to where you want to. It provides links to your most recent work." -%>
 
 <% unless current_user.swipecard_code? %>
   <%= alert(:warning) do %>
     You don't currently have a swipe-card configured. This is required to use some of the pipeline applications.
-    <%= link_to 'Click here to set one up now!', edit_profile_path(current_user), class: "alert-link" %>
+    <%= link_to 'Click here to set one up now', edit_profile_path(current_user), class: "alert-link" %>
     <%= help_link 'Further information', 'Add a swipecard to Sequencescape' %>
   <% end %>
 <% end %>
 
 <!-- Internal Links -->
-<% add :menu, 'Create Submission'      => new_submission_path %>
-<% add :menu, 'Create Bulk Submission' => bulk_submissions_path %>
-<% add :menu, 'Create Study'           => new_study_path %>
-<% add :menu, 'Create Project'         => new_project_path %>
-<% add :menu, 'Plate Summaries'        => plate_summaries_path %>
-<% add :menu, 'Create Custom Pools'    => new_pooling_path %>
+<% add :menu, 'Create Bulk Submission'  => bulk_submissions_path %>
+<% add :menu, 'Create Custom Pools'     => new_pooling_path %>
+<% add :menu, 'Create Project'          => new_project_path %>
+<% add :menu, 'Create Study'            => new_study_path %>
+<% add :menu, 'Create Submission'       => new_submission_path %>
+<% add :menu, 'Plate Summaries'         => plate_summaries_path %>
+<% add :menu, 'View Tube Rack Status'   => tube_rack_statuses_path %>
```

Comment:
I think the order before made more sense - submission and bulk submission next to each other